packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -2,9 +2,14 @@ import React, { Component } from 'react';
 import isNil from 'lodash/isNil';
 import classNames from 'classnames';
 import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { Themeable } from '../../types/theme';
+import { stylesFactory } from '../../themes';
+import { withTheme } from '../../themes/index';
 import Scrollbars from 'react-custom-scrollbars';
 
-interface Props {
+interface Props extends Themeable {
+  theme: GrafanaTheme;
```

Comment:
No need to add this as the `Props` interface extends Themeable. Mabye you should take the time to dive into what extending an Interface means before using it? https://www.typescriptlang.org/docs/handbook/interfaces.html#extending-interfaces

Reply:
Alright. would fix right away

Reply:
Great, but more important is to understand why `theme` isn't needed in this context. This is an opportunity to learn something very important.